view/base/requirejs-config.js

```diff
@@ -15,6 +15,9 @@ var config = {
             },
             'Adyen_Payment/js/view/payment/method-renderer/adyen-oneclick-method': {
                 'Signifyd_Connect/js/model/adyen-oneclick-method-mixin': true
+            },
+            "Holacash_Payment/js/view/payment/method-renderer/holacash_gateway": {
+                /Signifyd_Connect/js/model/holacash-cc-method-mixin': true,
```

Comment:
@jpaliwal why the key starts with `/` also seems it doesn't have `'`

Reply:
resolved